hhast-lint.json

```diff
@@ -22,5 +22,10 @@
         "Facebook\\HHAST\\DataProviderTypesLinter"
       ]
     }
-  ]
+  ],
+  "linterConfigs": {
+    "Facebook\\HHAST\\HHClientLinter": {
+      "ignore": [5562, 5583, 5568, 5607, 5609, 5611, 5616, 5618, 5624, 5628, 5630, 5639]
```

Comment:
perhaps this should be changed again to `dict<int, string>`, with the string as ignored comments, to make the config more maintainable?

e.g. 

```
"ignore": [5562 => "linter description: ignore reason" ]
```


Relatedly, what are these, and why are they ignored here, rather than automatically for all, or being fixed here?

Reply:
Probably we don't want to use "vanilla" JSON setting files because comments are supported.

Reply:
Added some an unused field as comment